main_routers/jukebox_router.py

```
@@ -0,0 +1,1156 @@
+            import_data = json.load(f)
+        
+        stats = {
+            "songsAdded": 0,
+            "songsMerged": 0,
+            "actionsAdded": 0,
+            "actionsMerged": 0,
+            "bindingsAdded": 0
+        }
+        
+        # 第一步：导入歌曲
+        for song_id, song in import_data.get("songs", {}).items():
```

Ruff (0.15.9)

[warning] Loop control variable `song_id` not used within loop body

(B007)